### PR TITLE
ident 3 pixel pushing

### PR DESCRIPTION
**Pull request by chrisnojima**
This fixes some layout issues in the tracker and profile

### CODE REVIEW

File: shared/profile/user/actions/index.js [hunk 1]
```diff
@@ -129,7 +129,6 @@ const DropdownButton = Kb.OverlayParentHOC(p => {
       <Kb.FloatingMenu
         closeOnSelect={true}
         attachTo={p.getAttachmentRef}
-        containerStyle={styles.floatingMenu}
```
Comment (chrisnojima) [inline shared/profile/user/actions/index.js]: not used

File: shared/profile/user/actions/index.js [hunk 2]
```diff
@@ -139,6 +138,8 @@ const DropdownButton = Kb.OverlayParentHOC(p => {
   )
 })
 
-const styles = Styles.styleSheetCreate({})
+const styles = Styles.styleSheetCreate({
+  chatIcon: {marginRight: Styles.globalMargins.tiny},
```
Comment (chrisnojima) [inline shared/profile/user/actions/index.js]: space between the icon and text

File: shared/tracker2/assertion/index.js [hunk 3]
```diff
@@ -232,7 +232,7 @@ const styles = Styles.styleSheetCreate({
   metaContainer: {flexShrink: 0, paddingLeft: 20 + Styles.globalMargins.tiny * 2 - 4}, // icon spacing plus meta has 2 padding for some reason
   site: {color: Styles.globalColors.black_20},
   stateIcon: {height: 17},
-  textContainer: {flexGrow: 1, marginTop: -1},
+  textContainer: {flexGrow: 1, flexShrink: 1, marginTop: -1},
```
Comment (chrisnojima) [inline shared/tracker2/assertion/index.js]: wrap long names

File: shared/tracker2/index.desktop.js [hunk 4]
```diff
@@ -137,7 +137,7 @@ const Tracker = (props: Props) => {
   return (
     <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} style={styles.container}>
       <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.header}>
-        <Kb.Icon type="iconfont-close" onClick={props.onClose} />
+        <Kb.Icon type="iconfont-close" onClick={props.onClose} style={styles.close} />
```
Comment (chrisnojima) [inline shared/tracker2/index.desktop.js]: extra padding on click area

File: shared/tracker2/index.desktop.js [hunk 5]
```diff
@@ -149,12 +149,14 @@ const Tracker = (props: Props) => {
           </Kb.Text>
           <Kb.Box2 direction="vertical" fullWidth={true} style={styles.avatarContainer}>
             <Kb.Box2 direction="vertical" style={styles.avatarBackground} />
-            <Kb.ConnectedNameWithIcon
-              onClick="profile"
-              username={props.username}
-              colorFollowing={true}
-              notFollowingColorOverride={Styles.globalColors.orange}
-            />
+            <Kb.Box2 direction="vertical" style={styles.nameWithIconContainer}>
```
Comment (chrisnojima) [inline shared/tracker2/index.desktop.js]: center this so the click area doesn't go full width

File: shared/tracker2/index.desktop.js [hunk 6]
```diff
@@ -223,19 +225,22 @@ const styles = Styles.styleSheetCreate({
     isElectron: {boxShadow: 'rgba(0, 0, 0, 0.15) 0px 0px 3px'},
   }),
   chatIcon: {marginRight: Styles.globalMargins.tiny},
-  container: {
-    backgroundColor: Styles.globalColors.white,
-    position: 'relative',
-  },
-  header: Styles.platformStyles({
+  close: {padding: Styles.globalMargins.tiny},
```
Comment (chrisnojima) [inline shared/tracker2/index.desktop.js]: extra padding around close button

File: shared/tracker2/index.desktop.js [hunk 7]
```diff
@@ -223,19 +225,22 @@ const styles = Styles.styleSheetCreate({
...
+  container: Styles.platformStyles({
     isElectron: {
       ...Styles.desktopStyles.windowDragging,
```
Comment (chrisnojima) [inline shared/tracker2/index.desktop.js]: lets you drag the whole window

**Comment by chrisnojima** [comment]
@keybase/react-hackers 
cc: @mmaxim @cecileboucheron

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/tracker2/assertion/index.js (hunk 3 above)
Comment (songgao) [reply shared/tracker2/assertion/index.js]: Do we need `lineClamp` on the component for mobile?

Comment (chrisnojima) [reply shared/tracker2/assertion/index.js]: no it wraps

**Pull request merged**